app-frontend/src/app/components/pages/project/project/index.html

```diff
@@ -1,37 +1,39 @@
-<div class="navbar light-navbar">
-  <div class="navbar-section">
-    <nav>
-      <a ui-sref="project.layers"
-      ng-class="{
-        active: ('project.layers' | includedByState)
-      }"
-      >Layers</a>
-    </nav>
-    <nav>
-      <a ui-sref="project.analyses"
-         ng-class="{
-                  active: ('project.analyses' | includedByState)
-                  }"
-      >Analyses</a>
-    </nav>
-  </div>
-  <div class="navbar-section secondary">
-    <nav>
-      <a>Smart Layers</a>
-    </nav>
-    <nav>
-      <a ui-sref="project.settings"
-         ng-class="{
-                  active: ('project.settings' | includedByState)
-                  }"
-      ><i class="icon-settings" title="Project Settings"></i></a>
-    </nav>
+<div ui-view="navbar-secondary">
+  <div class="navbar light-navbar">
+    <div class="navbar-section">
+      <nav>
+        <a ui-sref="project.layers"
+           ng-class="{
+                    active: ('project.layers' | includedByState)
+                    }"
+        >Layers</a>
+      </nav>
+      <nav>
+        <a ui-sref="project.analyses"
+           ng-class="{
+                    active: ('project.analyses' | includedByState)
+                    }"
+        >Analyses</a>
+      </nav>
+    </div>
+    <div class="navbar-section secondary">
```

Comment:
I think it would be better to hide this element at this point until it is implemented, which is similar to what we have been doing to other unimplemented features/functionalities for project layers.